iotdb/src/test/java/org/apache/iotdb/db/integration/IoTDBDeletionIT.java

```diff
@@ -147,6 +147,29 @@ public void testMerge() throws SQLException, InterruptedException {
     cleanData();
   }
 
+  @Test
+  public void testDelAfterFlush() throws SQLException {
+    Connection connection = DriverManager
+        .getConnection(Config.IOTDB_URL_PREFIX + "127.0.0.1:6667/", "root",
+            "root");
+    Statement statement = connection.createStatement();
+
+    statement.execute("SET STORAGE GROUP TO root.ln.wf01.wt01");
+    statement.execute("CREATE TIMESERIES root.ln.wf01.wt01.status WITH DATATYPE=BOOLEAN,"
+        + " ENCODING=PLAIN");
+    statement.execute("INSERT INTO root.ln.wf01.wt01(timestamp,status) "
+        + "values(1509465600000,true)");
+    statement.execute("INSERT INTO root.ln.wf01.wt01(timestamp,status) VALUES(NOW(), false)");
+
+    statement.execute("delete from root.ln.wf01.wt01.status where time < NOW()");
+    statement.execute("flush");
+    statement.execute("delete from root.ln.wf01.wt01.status where time < NOW()");
```

Comment:
You'd better add a query statement here for assert the result is expected.

Reply:
OK, though it is not the point of this test, which is to ensure the same bug won't recur.